Makefile-libs.Inc

```diff
@@ -27,6 +27,7 @@ ifeq ($(GOOS),windows)
 	platform := windows
 	export PATH := $(PATH):$(BIN_DIR)
 	CGO_LDFLAGS := $(CGO_LDFLAGS) -Wl,-Bstatic -lpthread -Wl,-Bdynamic
+	EXE := .exe
```

Comment:
I removed this from this file because it was not used anywhere and I didn't notice it's used in the Makefile that includes it. For this reason, in my opinion, `EXE` should be defined in the Makefile which uses it, not in this one.